spec.bs

```diff
@@ -5314,6 +5345,17 @@ An <dfn>auction report info</dfn> is a [=struct=] with the following [=struct/it
 
 </dl>
 
+<h3 id=k-anonymity-records>K-Anonymity Records</h3>
+A <dfn>k-anonymity record</dfn> is a timestamped cache of the k-anonymity status
+for a given k-anonymity key. These records are stored in the [=user agent=]

```

Comment:
what's a k-anonymity key? It's used multiple times. Can we define it and link these "k-anonymity key" to the definition?

Reply:
done